Manito/Manito/Screens/Main/MainViewController.swift

```diff
@@ -208,59 +209,21 @@ final class MainViewController: BaseViewController {
                 self?.refreshControl.endRefreshing()
             }
         }
-        refreshControl.addAction(action, for: .valueChanged)
-        refreshControl.tintColor = .grey001
-        listCollectionView.refreshControl = refreshControl
+        self.refreshControl.addAction(action, for: .valueChanged)
+        self.refreshControl.tintColor = .grey001
+        self.listCollectionView.refreshControl = self.refreshControl
     }
     
     private func setupSkeletonView() {
-        listCollectionView.showAnimatedGradientSkeleton(usingGradient: .init(colors: [.grey003, .darkGrey002]), animation: skeletonAnimation, transition: .none)
+        self.listCollectionView.showAnimatedGradientSkeleton(usingGradient: .init(colors: [.grey003, .darkGrey002]), animation: skeletonAnimation, transition: .none)
     }
     
     private func stopSkeletonView() {
         self.listCollectionView.stopSkeletonAnimation()
         self.listCollectionView.hideSkeleton(reloadDataAfter: true, transition: .crossDissolve(0.5))
     }
-    
-    // MARK: - API
-    
-    private func requestCommonMission() {
-        Task {
-            do {
-                let data = try await mainService.fetchCommonMission()
-                if let commonMission = data?.mission {
-                    commonMissionView.mission.text = commonMission
-                }
-            } catch NetworkError.serverError {
-                print("serverError")
-            } catch NetworkError.clientError(let message) {
-                print("clientError:\(String(describing: message))")
-            }
-        }
-    }
-    
-    private func requestManittoRoomList() {
-        Task {
-            do {
-                let data = try await mainService.fetchManittoList()
-                
-                if let manittoList = data {
-                    rooms = manittoList.participatingRooms
-                    listCollectionView.reloadData()
-                    
-                    self.stopSkeletonView()
-                }
-            } catch NetworkError.serverError {
-                print("serverError")
-            } catch NetworkError.clientError(let message) {
-                print("clientError:\(String(describing: message))")
-            }
-        }
-    }
-    
-    // MARK: - func
 
-    private func newRoom() {
+    private func createNewRoom() {
         let alert = UIAlertController(title: "새로운 마니또 시작", message: nil, preferredStyle: UIAlertController.Style.actionSheet)
```

Comment:
p2;

여기도 줄을 바꾸고 `UIAlertController.Style.actionSheet` 이 부분을 `.actionSheet`요렇게 적는건 어떨까요 ??
가능하다면 "새로운 마니또 시작" 부분도 TextLiteral로 분리가 되면 좋겠습니다 !

```suggestion
        let alert = UIAlertController(title: "새로운 마니또 시작",
                                      message: nil,
                                      preferredStyle: .actionSheet)
```

Reply:
[5b78e36](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/375/commits/5b78e369fc833d0907b44a105c6953832ed757cf)
반영했습니다.